web/core/src/jsMain/kotlin/androidx/compose/web/css/StyleSheet.kt

```diff
@@ -39,17 +40,21 @@ class CSSRulesHolderState : CSSRulesHolder {
  * ```
  */
 open class StyleSheet(
-    private val rulesHolder: CSSRulesHolder = CSSRulesHolderState()
+    private val rulesHolder: CSSRulesHolder = CSSRulesHolderState(),
+    val usePrefix: Boolean = true,
```

Comment:
I didn't find/see any cases when `usePrefix = false`.
What's the idea behind this parameter? When would one want to pass `false`?

Reply:
It's for the cases when user want integrate with some third party lib and it's classes.